src/actions/Setting.js

```diff
@@ -25,9 +25,10 @@ export function getOrgKubeconfigValidity(orgId) {
   ).get();
 }
 
-export function setOrgKubeconfigValidity(orgId, params) {
+export function setOrgKubeconfigValidity(orgId, accId, params) {
```

Comment:
why are we passing in acc id for org kubeconfig validity ?

Reply:
So that if settings are made w.r.t to accounts as well, this can be used. But since now, we are dealing with org settings we can get rid of this.

Reply:
reverted